earthfile2llb/converter.go

```diff
@@ -672,11 +673,21 @@ func (c *Converter) internalRun(ctx context.Context, args []string, secretKeyVal
 		}
 	}
 
+	finalOpts = append(finalOpts,
+		llb.AddMount("/usr/bin/earth_debugger",
+			llb.Image(c.debuggerImage, llb.MarkImageInternal, llb.ResolveModePreferLocal, llb.Platform(llbutil.TargetPlatform)),
```

Comment:
I have an idea... if we host-bind the debugger binary too, then we wouldn't bust the cache if the debugger binary changes. What do you think? (This can be a separate PR though)

Reply:
that would also make distributing the debugger binary easier. Currently we have to manually push it to docker hub, this would greatly improve that process.